notifications.rb

```diff
@@ -33,7 +33,7 @@ class Notifications < Sinatra::Application
   post '/event/new?' do
     event_info = GoogleService.create_event(token: params[:token],
                                             refresh_token: params[:refresh_token],
-                                            calendar: params[:calendar],
+                                            calendar_id: params[:calendar_id],
```

Comment:
this change will allow to create `events` for the `user` even if the `name` of the `calendar` is changed after signing in